lib/twingly/livefeed/version.rb

```diff
@@ -0,0 +1,5 @@
+module Twingly
+  module LiveFeed
+    VERSION = "5.0.1"
```

Comment:
> The API:s look very similar (at least the responses) so I doubt there will be any difference in dependencies, so one gem would work. Should we just move the VERSION constant to the Twingly module instead? It's only used in the gemspec anyway.

I think we should move it, not sure about `Twingly::VERSION` though, since other gems might use that namespace as well. How about something like `Twingly::BlogClient::Version` or something like that?